libs/generic-view/ui/src/lib/texts/highlight-text.tsx

```diff
@@ -0,0 +1,47 @@
+/**
+ * Copyright (c) Mudita sp. z o.o. All rights reserved.
+ * For licensing, see https://github.com/mudita/mudita-center/blob/master/LICENSE.md
+ */
+
+import React, { ReactElement, useCallback } from "react"
+import { APIFC } from "generic-view/utils"
+import { HighlightTextConfig, HighlightTextData } from "generic-view/models"
+
+export const HighlightText: APIFC<HighlightTextData, HighlightTextConfig> = ({
+  data,
+  config,
+}) => {
+  // TODO: Implement more modes when needed
```

Comment:
it could apply to any part of our codebase. Let’s face it, there’s room for improvement everywhere 😅 😉 
```suggestion
```

Reply:
Fixed in [c7a96aa](https://github.com/mudita/mudita-center/pull/2191/commits/c7a96aaf54c1a90a5ac7e6fd86351494900d1a6f)